solidity/contracts/deposit/DepositLiquidation.sol

```diff
@@ -74,7 +74,7 @@ library DepositLiquidation {
         if (_d.auctionTBTCAmount() == 0) {
             // we came from the redemption flow
             _d.setLiquidated();
-            _d.redeemerAddress.transfer(_seized);
+            _d.redeemerAddress.call.value(_seized)("");
```

Comment:
Feels like we should do this at the end of the function, per checks-effects-interactions, no?

Reply:
we `setLiquidated` before transferring, preventing reentrancy. 

>  at the end of the function

Assuming you mean above `setFraudLiquidationInProgress`. I'm fine with that, although the values are unrelated.